datadog_checks_base/datadog_checks/checks/openmetrics/mixins.py

```diff
@@ -0,0 +1,747 @@
+        config['rate_metrics'].extend(self._extract_rate_metrics(instance.get('type_overrides', [])))
+
+        # `_metrics_wildcards` holds the potential wildcards to match for metrics
+        config['_metrics_wildcards'] = None
+
+        # `prometheus_metrics_prefix` allows to specify a prefix that all
+        # prometheus metrics should have. This can be used when the prometheus
+        # endpoint we are scrapping allows to add a custom prefix to it's
+        # metrics.
+        config['prometheus_metrics_prefix'] = instance.get('prometheus_metrics_prefix', default_instance.get('prometheus_metrics_prefix', ''))
+
+        # `label_joins` holds the configuration for extracting 1:1 labels from
+        # a target metric to all metric matching the label, example:
+        # self.label_joins = {
+        #     'kube_pod_info': {
+        #         'label_to_match': 'pod',
+        #         'labels_to_get': ['node', 'host_ip']
+        #     }
+        # }
+        config['label_joins'] = default_instance.get('label_joins', {})
+        config['label_joins'].update(instance.get('label_joins', {}))
+
+        # `_label_mapping` holds the additionals label info to add for a specific
+        # label value, example:
+        # self._label_mapping = {
+        #     'pod': {
+        #         'dd-agent-9s1l1': [("node","yolo"),("host_ip","yey")]
+        #     }
+        # }
+        config['_label_mapping'] = {}
+
+        # `_active_label_mapping` holds a dictionary of label values found during the run
+        # to cleanup the label_mapping of unused values, example:
+        # self._active_label_mapping = {
+        #     'pod': {
+        #         'dd-agent-9s1l1': True
+        #     }
+        # }
+        config['_active_label_mapping'] = {}
+
+        # `_watched_labels` holds the list of label to watch for enrichment
+        config['_watched_labels'] = set()
+
+        config['_dry_run'] = True
+
+        # Some metrics are ignored because they are duplicates or introduce a
+        # very high cardinality. Metrics included in this list will be silently
+        # skipped without a 'Unable to handle metric' debug line in the logs
+        config['ignore_metrics'] = []
+
+        # If you want to send the buckets as tagged values when dealing with histograms,
+        # set send_histograms_buckets to True, set to False otherwise.
+        config['send_histograms_buckets'] = is_affirmative(instance.get('send_histograms_buckets',
+                                                           default_instance.get('send_histograms_buckets', True)))
+
+        # If you want to send `counter` metrics as monotonic counts, set this value to True.
+        # Set to False if you want to instead send those metrics as `gauge`.
+        config['send_monotonic_counter'] = is_affirmative(instance.get('send_monotonic_counter',
+                                                          default_instance.get('send_monotonic_counter', False)))
```

Comment:
Yes, let's do this as well. Originally, I had made the default to `False` because the default was `False` in the mixin:
https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/checks/prometheus/mixins.py#L447